project.clj

```diff
@@ -208,7 +207,7 @@
                                                ;; brings in its own version, and older versions of
                                                ;; lein depend on clojure 1.6.
                                                [org.clojure/clojure nil]
-                                               [puppetlabs/puppetserver ~ps-version :exclusions [puppetlabs/jruby-deps]]
+                                               [puppetlabs/puppetserver ~ps-version]
```

Comment:
Can we get rid of this line complete?

Reply:
+1

Reply:
I'll try it, however, it was still present before we added the exclusion, see https://github.com/puppetlabs/puppetserver/commit/59ebc8cf6e5834a14822746578d8c1194fa65132 which is why I left it in.

Reply:
I believe https://github.com/puppetlabs/puppetserver/pull/319 is where it was added

Reply:
Not that there's any information in that ticket why this is necessary. However looking at PDB they have the same line in their [project.clj](https://github.com/puppetlabs/puppetdb/blob/master/project.clj#L180-L183) with a ticket number that actually explains this pretty well.

tl;dr
Everyone ends the ticket believing this should no longer be necessary and Jeremy includes it in [SERVER-1053](https://tickets.puppetlabs.com/browse/SERVER-1053) in a long list of things to look at when automating Puppet Server's release (ie removing this instead of updating the cljupdatedeps script). We closed Jeremy's ticket after a re-orged because no one knew what issues he was referencing in his list any more.

Reply:
Re-opened [EZ-35](https://tickets.puppetlabs.com/browse/EZ-35) for allowing us to remove this line.